magicctapipe/resources/config.yaml

```diff
@@ -279,9 +279,22 @@ dl2_to_dl3:
     source_dec: null  # used when the source name cannot be resolved
     
 cloud_correction:
-    specify_cloud: yes
-    base_height: "5900. m" 
-    thickness: "1320. m"
-    vertical_transmission: 1.0
+    max_gap_lidar_shots: "900 s"
+    lidar_report_file: "../../resources/lidar_reports_clouds_ST.03.16.yaml"
     number_of_layers: 10
 
+    additional_lst_cleaning:
+        picture_thresh: 10
```

Comment:
the purpose of redoing the cleaning is to raise the thresholds by some factor, and the same factor should be applied to LST and MAGIC. So, either the factor is determined automatically in the script (but this can be messy because then one needs to process the simulations for each run even if changes are small, or you specify in the input card only a single multiplication factor and this factor is applied to picture_thresh, boundary_thresh of both MAGIC and LST, and rest of the settings should be taken from default config or from the file itself (at least for MAGIC, because for LST this can be more tricky)